taxonomies/labels.txt

```diff
@@ -3717,13 +3717,10 @@ country:en:Belgium
 image:en:biogarantie-bel.90x90.svg
 
 <en:EU organic
-en:EG-Öko-Verordnung
```

Comment:
shouldn't we keep it ? It might break some external stuff

Reply:
It's covered by xx:EG-Öko-Verordnung, external tools may have to add support for xx: synonyms if they don't have it yet though.